misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -96,4 +98,8 @@ class AccessInterceptor private constructor(
     private inline fun <reified T : Annotation> Action.hasAnnotation() =
         function.annotations.any { it.annotationClass == T::class }
   }
+
+  companion object {
+    val log = getLogger<AccessInterceptor>()
```

Comment:
We tend to use `logger` as the val instead of `log`.

Reply:
fixed